module-config.yaml

```diff
@@ -1,6 +1,6 @@
 name: kyma-project.io/module/template-operator
 channel: regular
-version: v1.0.0
+version: 0.1.0
```

Comment:
why change this version?

Reply:
Shouldn't this be aligned with the release? First release will be 0.1.0

Reply:
ok, this might affect LM e2e testing, will follow this, let's merge it.